<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugForm</class>
 <widget class="QWidget" name="DebugForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="pwd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmPwd">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>输入密码</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>981</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QDoubleSpinBox" name="tempThres">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>0.900000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="testFindPic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>②测试 读取模板 匹配</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getGameWin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>①测试 点击按钮 获取图像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testPress2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>测试按2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="test1">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>ESC 活动</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testPressM">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>测试按M</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testFindColor">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>②测试 颜色匹配</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="colorEditText">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="testFastSwitch">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>开始速切人战斗</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testNoSwitch">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>开始不切人战斗</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testRebootGame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>测试重启游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="activateBtn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>激活窗体 不断截图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deactivateBtn">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>断开激活</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="capImgWaitMs">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>80</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="suffix">
     <string>    ms</string>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>80</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>截图间隔</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>截图保存路径</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="saveImgPath">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>331</width>
      <height>64</height>
     </rect>
    </property>
    <property name="plainText">
     <string>c:/mchtpic.bmp</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="isActivate">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>90</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>截图中?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scrCap">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>210</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>截图一次
CTRL+P</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
